app/services/data/get-claim-list-for-search.js

```diff
@@ -0,0 +1,37 @@
+const config = require('../../../knexfile').intweb
+const knex = require('knex')(config)
+const moment = require('moment')
+
+module.exports = function (query, offset, limit) {
+  query = `%${query}%` // wrap in % for where clause
+  return knex('Claim')
+    .join('Eligibility', 'Claim.EligibilityId', '=', 'Eligibility.EligibilityId')
```

Comment:
Can remove this join as it is unneccessary.  The only thing we would need from Eligibility is the Reference, which is already on the Claim record.